modules/dicom_archive/templates/menu_dicom_archive.tpl

```diff
@@ -107,14 +107,22 @@
     {/if}
     </div>
 
-    <table border="0" valign="bottom" width="100%">
-    <tr>
-        <!-- title -->
-        <td class="controlPanelSection"></td>
-        <!-- display pagination links -->
-        <td align="right">{$page_links}</td>
-    </tr>
-    </table>
+    <div id="pageLinks" class="row col-md-3 col-sm-12 col-md-push-9">
+    </div>
+    <script>
+    var pageLinks = RPaginationLinks(
+        {
+                    RowsPerPage : 50,
```

Comment:
Oops.. I thought I did (I specifically assigned it to a smarty variable so I could..). The other places are pure-JS, so they don't have access to the config, but I can change it here.